syntax = "proto3";

package api.v1;

service GreeterService {
  rpc SayHello(HelloRequest) returns (HelloResponse) {}
}

message HelloRequest { string name = 1; }

message HelloResponse { string message = 1; }
